packages/client/src/arpa_reporter/components/StandardForm.vue

```diff
@@ -59,7 +59,7 @@
       <div class="col-sm-2" />
       <div class="col-sm-10">
         <button
-          class="btn btn-primary"
+          class="btn usdr-btn-primary"
```

Comment:
Note to reviewer: before/after screenshots

## Before
![image](https://github.com/user-attachments/assets/0f1485f3-7e4a-4c51-999e-2dfe55e360c5)


## After
![image](https://github.com/user-attachments/assets/819097ed-c41d-47a1-9825-9c887f5f6f8c)